include/mbedtls/config.h

```diff
@@ -3036,6 +3036,21 @@
  */
 #define MBEDTLS_PKCS5_C
 
+/**
```

Comment:
After adding an option in `config.h`, you need to run `scripts/generate_query_config.pl` and `scripts/generate_features.pl`. Or, more simply, run `tests/scripts/check-generated-files.sh -u` to update whatever needs updating.

Reply:
> After adding an option in `config.h`, you need to run `scripts/generate_query_config.pl` and `scripts/generate_features.pl`. Or, more simply, run `tests/scripts/check-generated-files.sh -u` to update whatever needs updating.

Sorry, this comment is not clear to me as I have done this as part of the following commits:

mbedtls: add pkcs7 in generate_errors.pl 3c49d62
mbedtls: updates features file for pkcs7 6e0d3fd 

What did I miss ?

Thanks & Regards,
     - Nayna